### DISCUSSION TITLE
[COMPLETED] Proposal: Detaching as a fork of Tilt Brush on GitHub

### DISCUSSION THREAD

**Discussion by mikeskydev** [icosa-gallery/open-brush#320] [Policy]
Open Brush is nearing 500 stars and has multiple contributors 🎉

However, regular contributors have noticed that GitHub's built in search tool does not function with forked repositories both code search and searching the platform generally, nor does it display a list of contributors on the sidebar. This may be hindering new contributor discovery.

GitHub's reasoning for this is outlined here: https://docs.github.com/en/search-github/searching-on-github/searching-in-forks, but in this case while Tilt Brush still has more stars, it's been in read-only mode since the beginning.

There is a method of 'detaching' a fork from it's parent, with the option to bring along it's own child forks. This is supposedly achieved by contacting support: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#commit-was-made-in-a-fork 

My proposal is that we detach Open Brush as a fork at 500 stars for more visibility.

In my opinion we should also bring along the child forks, as we're the most forked repository of Tilt Brush too.

My only concern is people looking for activity in Tilt Brush's own forks who may no longer notice Open Brush and it's healthy community in the list of forks.

**Comment by TimAidley** [comment]
I support this proposal. I think it's worth getting out from under Tilt Brush's shadow. 

I realize that we risk missing folks who search for Tilt Brush, but that's already the case anyway. I think as development on Open Brush continues it will become people's preferred version anyway.

**Comment by andybak** [comment]
Yeah. It's annoying that this is an either/or choice because of weaknesses in Github's design - but this definitely feels like the lesser of the two evils.

**Comment by mikeage** [comment]
I think we should add a link to the original repo in the top of the README, but I'm in favor, given the popularity of OB vs the number of people who will find us from TB.

**Reply by mikeskydev** [reply]
Yep this is something I'd like to do, ensure there's *some* sort of back reference. Would be nice to somehow have a pointer on this list however https://github.com/googlevr/tilt-brush/network/members

**Comment by andybak** [comment]
I vote for doing this now. I keep accidentally submitting PRs to the original repo and losing all my carefully typed notes!

**Reply by mikeskydev** [reply]
I keep doing that too 😅 I'm happy to bring this forward, 500 stars was arbitrary milestone anyways.

**Comment by mikeskydev** [comment]
This has now been actioned and a request has been submitted to GitHub support. Thank you all for your input!
